cli/src/commands/tui/ui/logs.rs

```diff
@@ -73,7 +71,7 @@ impl LogsWidget {
         }
 
         render_log_entries(frame, logs_area, &mut app.state);
-        render_scrollbar(frame, logs_area, &mut app.state.logs_vertical_scroll_state);
+        // render_scrollbar(frame, logs_area, &mut app.state.logs_vertical_scroll_state);
```

Comment:
Whats the reason hiding the scrollbar?

Reply:
we are using the custom scrolling implementation, so the built-in scrollbar doesn't work well with that.